css/templates/lune/page-vars.css

```diff
@@ -0,0 +1,70 @@
+:root {
+  --content-max-width: 100%;
+
+  --z1: 1; /* body */
+  --z2: 2; /* overlay */
+  --z3: 3; /* sidebar mask */
+  --z4: 4; /* sidebar */
+  --z5: 5; /* header */
+
+  --layout-header-height-mobile: 4.375rem; /* 70px */
+  --layout-header-height-desktop: 5.75rem; /* 92px */
+
+
+
+  /* Type */
+  --font-family: "Lato", -apple-system, BlinkMacSystemFont, "Helvetica Neue", "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", Arial, sans-serif;
+  --font-weight-black: 900;
+
+  /* Responsive headings type scale */
+  --h00: 6.875rem; /* 110px */
+  --h0: 5rem; /* 80px */
+  --h1: 4.0625rem; /* 65px */
+  --h2: 3.125rem; /* 50px */
+  --h3: 2.1875rem; /* 35px */
+  --h4: 1.375rem; /* 22px */
+  --h5: 1.125rem; /* 18px */
+  --h6: .875rem; /* 14px */
+  --h7: .75rem; /* 12px */
+
+
+  /* Letter spacing */
+  --letter-spacing-1: .02em;
```

Comment:
Following on a previous comment, we're using ems so the letter spacing will scale proportionally to the current font size of the element. This will save us from creating further variations and adding unnecessary breakpoints to switch them out